scopes/dependencies/yarn/yarn.docs.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Enables the use of Yarn 2 in a Bit workspace
+labels: ['pnpm', 'package manager']
+---
+
+Yarn is a Bit aspect that enables the use of the Yarn v2 package manager in a Bit workspace.  
+The yarn aspect is used **indirectly** by the [Dependency Resolver](https://bit.dev/dependencies/pnpm).
+
+## Quickstart
+
+To start using the yarn aspect as a package manager for your workspace, set the Dependency Resolver `packageManager` property to 'yarn'.
+
+```json
+{
+  "teambit.dependencies/dependency-resolver": {
+    "packageManager": "teambit.dependencies/yarn"
+  }
+}
+```
+
+> Package manager are only configurable at th workspace configuration root-level.
```

Comment:
mention that bit uses the programmatic api of yarn 2
maybe mention that yarn2 is not production yet